### ISSUE TITLE
Error when compiling blueprint with rollup

### ISSUE THREAD

**Issue by ahutchings** [palantir/blueprint#2221]
**Blueprint version:** @blueprintjs/core@2.0.0-rc.2

I have a library that depends on `@blueprintjs/core`. When compiling the library via rollup, I see the following error:

```bash
$ rollup -c

src/index.js → dist/index.cjs.js, dist/index.esm.js...
[!] Error: Cannot call a namespace ('classNames')
node_modules/@blueprintjs/core/lib/esm/components/forms/controls.js (33:24)
31:     Control.prototype.renderControl = function (type, typeClassName, inputRef) {
32:         if (inputRef === void 0) { inputRef = this.props.inputRef; }
33:         var className = classNames(Classes.CONTROL, typeClassName, (_a = {},
                            ^
34:             _a[Classes.DISABLED] = this.props.disabled,
35:             _a[Classes.INLINE] = this.props.inline,
```

I believe this is due to the way `classnames` is imported with `
import * as classNames from "classnames";`. I found [this issue in the rollup repo](https://github.com/rollup/rollup/issues/670#issuecomment-223046229) which suggests changing the import to be `import classNames from "classnames";`.

Would you accept a PR to update the className import statements to be compatible with rollup?

**Comment by giladgray** [comment]
Duplicate of #2144. Will be fixed by PR #2200. Cheers.

**Comment by ahutchings** [comment]
Oops, sorry for the dupe. Thanks!
